src/ConicBenchmarkUtilities.jl

```diff
@@ -4,6 +4,21 @@ module ConicBenchmarkUtilities
 
 using GZip
 
+if VERSION < v"0.7.0-"
+    import Compat: undef
+    import Compat: @warn
+end
+
+if VERSION > v"0.7.0-"
+    using SparseArrays
+    using LinearAlgebra
+    # this is required because findall return type changed in v0.7
+    function SparseArrays.findnz(A::AbstractMatrix)
```

Comment:
It's better to update the code to work with the new return types so that later we can just delete the 0.6 compatibility blocks and have clean 0.7/1.0 code.

Reply:
Agreed this would be preferable.  I tried to do it that way for about an hour, but in the end I think it is impossible or requires a much deeper knowledge of sparse matrices in Julia than I have.

The root of the challenge is that the semantics of `findall` have also changed in v0.7.  For the tests in this package, the v0.6 `findall` returns a list of integers (not clear how ints map back to matrix cells) and the v0.7 `findall` returns a list of Cartesian data types.  So the workaround code proposed for v0.7 is incompatible with v0.6.  Using Compat on `findall` to bring the v0.7 semantics into v0.6 is also no successful.